packages/dart_frog/test/src/serve_test.dart

```diff
@@ -90,6 +90,48 @@ void main() {
       });
     });
 
+    group('shared', () {
+    test(
```

Comment:
Can we run the formatter here? It doesn't seem to have proper indentation?

Reply:
Yeah that is weird